### ISSUE TITLE
Unable to set Aria-Required for TextField

### ISSUE THREAD

**Issue by shgopala1** [OfficeDev/office-ui-fabric-react#11585]
### Describe the issue:

There is no property in ITextFieldProps for Aria-Required. Even the current example specified in Office UI Fabric page with Required(*) field doesn't have Aria-Required set.

**Comment by jdhuntington** [comment]
Hi @shgopala1 - Since the addition of the `required` attribute in more recent versions of HTML, we opt to set that instead. `aria-required` is redundant for all browsers we support, so it is not part of the props.

reference: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-required_attribute

**Comment by shgopala1** [comment]
@jdhuntington So is there a way to specify required without the red (*) asterik appearing next to the text field or label.

**Comment by jdhuntington** [comment]
@shgopala1 Yes, there is a way.
It requires overriding the props passed to `onRenderLabel` so that the label doesn't know that the control is required.

Have a look at this example: https://codepen.io/jdh-msft/pen/vYEdxZK?editors=0010
